backend/src/scrape/agencyType.js

```diff
@@ -0,0 +1,18 @@
+const agenciesType = [
+    "Government",
+    "Multinational",
+    "Commercial",
+    "Educational",
+    "Private",
+];
+
+function agencyType(agencyID, agenciesResolutionTable = agenciesType) 
+{
+    if(agencyID > 0 && agencyID - 1 < agenciesResolutionTable.length) 
```

Comment:
For future reference (not suggesting a change, this is absolutely fine!): a ternary expression would shine here IMO ;)